Docs: What's New content for v9.3

chri2547 opened the pull request:
This PR describes what's new in version 9.3.

chri2547 committed:
initial content for What's New 9.3

brendamuir commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+Add a screenshot to this enablement material folder and link to it here
+
```
Let's delete this line?

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,74 @@
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+
```
```suggestion
Add a screenshot to this enablement material folder and link to it here.
```

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

eleijonmarck committed:
docs: add conflict cli tool

chri2547 committed:
makes prettier

eleijonmarck committed:
remoed the conflict users tool in favor of not having it completely out yet

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+# What’s new in Grafana v9.3 (Beta)
+
+Welcome to Grafana v9.3. If you’d prefer to dig into the details, check out the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```
```suggestion
Welcome to Grafana v9.3. This document provides an overview of what's new in this release. For more detail, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

brendamuir replied:
How about "Welcome to Grafana 9.3. Read the What's New to learn all about the latest release."

Maybe a little more chatty and less formal, directly addressing the user...?

(eventually here we can add: prepare for upgrade, upgrade..?

chri2547 replied:
I like it, thanks.

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+## Public Dashboards - Audit Table
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced
```
```suggestion
This experimental feature is available in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced
+
+We have a new feature as part of our Public Dashboard efforts. We have introduced a new menu item under Dashboards → Public Dashboards. This new view is a list of all of the public dashboards in your Grafana instance. From here you can navigate to the underlying dashboard, see if it is enabled, get a quick link out to the public version of the dashboard, or get quick access to the configuration.  ...
```
```suggestion
A new menu item in this release under **Dashboards > Public Dashboards** enables you to view a list of all of the public dashboards in your Grafana instance. From this list, you can:
- Navigate to the underlying dashboard, if you have view access
- Determine if a dashboard is enabled
- Link out to the public version of the dashboard
- Access the dashboard configuration (if you have RBAC `Dashboard Public` or Admin basic privileges)
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+## Transformations - Partition by values
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced
```
```suggestion
This experimental feature is available in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced
+
+This new transformation can help eliminate the need for multiple queries to the same datasource with different WHERE clauses when graphing multiple series. Consider a metrics SQL table with the following data:
```
```suggestion
Instead of writing multiple queries, each with a different WHERE clause, the partition by values feature enables you to issue one query and split the results by unique (enum) values from one or more columns (fields) of your choosing.

Consider a metrics SQL table that includes the following data:
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+| 2022-10-20 01:00:00 | US     | 1327  |
+| 2022-10-20 01:00:00 | EU     | 912   |
+
+Prior to v9.3. if you wanted to plot a red trendline for US and a blue one for EU in the same TimeSeries panel, you would likely have to split this into two queries:
```
```suggestion
Before v9.3, to plot a red trend line for the US and a blue trend line for the EU in the same TimeSeries panel, you would likely have to split this into two queries:
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
````diff
@@ -0,0 +1,75 @@
+   SELECT Time, Value FROM metrics WHERE Time > ‘2022-10-20’ AND Region=’EU’
+```
+
+This also requires you to know ahead of time which regions actually exist in the metrics table.
````
```suggestion
This approach also requires you to know ahead of time which regions exist in the metrics table.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+This also requires you to know ahead of time which regions actually exist in the metrics table.
+
+With the Partition by values transformer, you can now issue a single query and split the results by unique (enum) values from one or more columns (fields) of your choosing. In this case, Region.
```
```suggestion
With the partition by values transformer, you can issue a single query and split the results by unique (enum) values from one or more columns (fields) of your choosing. In this case, Region.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,75 @@
+
+Welcome to Grafana v9.3. If you’d prefer to dig into the details, check out the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## What's New feature template
+
+Use this template to add your what's new section.
+
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+Add a screenshot to this enablement material folder and link to it here.
```
```suggestion
```

owensmallwood committed:
Adds note about pubdash annotations support in 9.3-beta notes

owensmallwood committed:
puts audit table photo back in right section

owensmallwood commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,81 @@
+
+[image public-dashboard-audit-table.png]
+
+## Public Dashboards - Annotations
```
@chri2547 Added blurb about annotations suppport

chri2547 replied:
super, thank you!

mgyongyosi committed:
Docs: Add OAuth improvements to what's new in 9.3 (#58756)

* Add OAuth improvements to what's new in v9.3

* Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v9-3.md

* Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Mitch Seaman <mjseaman@users.noreply.github.com>
Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

eleijonmarck committed:
Docs: Add conflict cli tool (#58827)

docs: add conflict cli tool

IevaVasiljeva committed:
docs: add Terraform updates for What's New in Grafana 9.3 (#58858)

* Terraform updates for Grafana 9.3

* Remove empty line

* linting

mgyongyosi committed:
Docs: Update OAuth improvements section of what's new in 9.3  (#59045)

Update OAuth improvement docs for 9.3

IevaVasiljeva committed:
fix incorrect version number

brendamuir committed:
Adds Alerting whats new entries

nmarrs committed:
Merge remote-tracking branch 'origin' into docs/whats-new-in-9.3

nmarrs committed:
Add edge squad 9.3 whats new information

nmarrs commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,183 @@
+
+Add a screenshot to this enablement material folder and link to it here.
+
+## Geomap panel
+
+Generally available in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced
+
+We have added a new alpha layer type in Geomap called photo layer. This layer enables you to render a photo at each data point. To learn more about the photo layer and the geomap panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/geomap/#photos-layer-alpha" >}}).
+
+{{< figure src="/static/img/docs/geomap-panel/geomap-photos-9-3-0.png" max-width="750px" caption="Geomap panel photos layer" >}}
+
+## Canvas panel
+
+Available in beta in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced
+
+We have added icon value mapping support. This enables you to dynamically set which icon to display based on your data. To learn more about the Canvas panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
+
+{{< video-embed src="/static/img/docs/canvas-panel/canvas-icon-value-mapping-support-9-3-0.mp4" max-width="750px" caption="Canvas panel icon value mapping support" >}}
```
Hi @chri2547, I've added info for 9.3 release from edge squad. Thanks!

[Geomap image added here](https://github.com/grafana/website/pull/10847)

[Canvas video added here](https://github.com/grafana/website/pull/10852)

gamab committed:
Docs: Update whats-new 9.3 with auth related news (#59093)

* LDAP role mapping improvements

* RBAC list token's permissions

* Azure force_use_graph_api

* Reorder

* LDAP uniformize

* Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Ieva <ieva.vasiljeva@grafana.com>

* Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Ieva <ieva.vasiljeva@grafana.com>

* Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Ieva <ieva.vasiljeva@grafana.com>

Co-authored-by: Ieva <ieva.vasiljeva@grafana.com>

chri2547 committed:
adds new nav and new language to what's new

mjseaman committed:
Add report zoom to What's New in 9.3 (#59345)

* Add report zoom to What's New in 9.3

Still needs a link to docs, a double-check on the name of the feature toggle, and a screenshot.

* Update whats-new-in-v9-3.md

* Apply suggestions from code review

* makes prettier

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>
Co-authored-by: Chris Moyer <chris.moyer@grafana.com>

mjseaman committed:
add report zoom image

mjseaman committed:
fix typo

mjseaman committed:
fix link to upload images in template description

mjseaman committed:
Copy edits

mjseaman committed:
copy and format updates

chri2547 committed:
Apply suggestions from code review

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,254 @@
+
+# What’s new in Grafana v9.3 (Beta)
+
+Welcome to Grafana v9.3. If you’d prefer to dig into the details, check out the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```
```suggestion
Welcome to Grafana 9.3. Read What's New to learn all about the latest release. For more detail, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,254 @@
+
+Welcome to Grafana 9.3. Read What's New to learn all about the latest release. For more detail, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## What's New feature template
+
+Use this template to add your what's new section.
+
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+Add a screenshot to the [website/static/static/img/docs](https://github.com/grafana/website/tree/master/static/static/img/docs) folder and link to it here.
```
```suggestion
```

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,245 @@
+
+Welcome to Grafana 9.3. Read What's New to learn all about the latest release. For more detail, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+
```
```suggestion
```

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+To try out Grafana’s new navigation, enable the `topnav` feature toggle. If you are a Cloud Advanced customer, open a ticket with our support team and we will enable it for you.
+
+**Note:** The Grafana documentation has not yet been updated to reflect changes to the navigation.
```
```suggestion
**Note:** The Grafana and Grafana Cloud documentation has not yet been updated to reflect changes to the navigation.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Generally available in all editions of Grafana
+
+We have added 4 new languages to Grafana: Spanish, French, German and Simplified Chinese.
```
```suggestion
We have added four new languages to Grafana: Spanish, French, German, and Simplified Chinese.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+We have added 4 new languages to Grafana: Spanish, French, German and Simplified Chinese.
+
+With millions of users across the globe, Grafana has a global footprint. In order to make it accessible to a wider audience, we have taken the first steps in localizing key workflows. You can now set Grafana’s language for the navigation, viewing dashboards, and a handful of settings. This will cover the main activities a Viewer would perform within Grafana.
```
```suggestion
With millions of users across the globe, Grafana has a global footprint. In order to make it accessible to a wider audience, we have taken the first steps in localizing key workflows. You can now set Grafana’s language for the navigation, viewing dashboards, and some settings. This will cover the main activities a Viewer performs within Grafana.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+With millions of users across the globe, Grafana has a global footprint. In order to make it accessible to a wider audience, we have taken the first steps in localizing key workflows. You can now set Grafana’s language for the navigation, viewing dashboards, and a handful of settings. This will cover the main activities a Viewer would perform within Grafana.
+
+Read more about configuring the [default language for your organization](https://grafana.com/docs/grafana/latest/administration/organization-preferences/) and [updating your profile](https://grafana.com/docs/grafana/latest/administration/user-management/user-preferences/) in our documentation.
```
```suggestion
Read more about configuring the [default language for your organization]({{< relref "../administration/organization-preferences/" >}}) and [updating your profile]({{< relref "../administration/user-management/user-preferences/" >}}) in our documentation.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Generally available in all editions of Grafana
+
+We have added a new alpha layer type in Geomap called photo layer. This layer enables you to render a photo at each data point. To learn more about the photo layer and the geomap panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/geomap/#photos-layer-alpha" >}}).
```
```suggestion
We have added a new alpha layer type in Geomap called photo layer. This layer enables you to render a photo at each data point. To learn more about the photo layer and the geomap panel, refer to [Photo layers]({{< relref "../panels-visualizations/visualizations/geomap/#photos-layer-alpha" >}}).
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Canvas is a new panel that combines the power of Grafana with the flexibility of custom elements. Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren’t possible with standard Grafana panels, all within Grafana’s UI. If you’ve used p ...
+
+In Grafana v9.3 We have added icon value mapping support to the Canvas panel. This enables you to dynamically set which icon to display based on your data. To learn more about the Canvas panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
```
```suggestion
In Grafana v9.3 We have added icon value mapping support to the Canvas panel. This enables you to dynamically set which icon to display based on your data. To learn more about the Canvas panel, refer to  [Canvas]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
```

nmarrs replied:
```suggestion
In Grafana v9.3 we have added icon value mapping support to the Canvas panel. This enables you to dynamically set which icon to display based on your data. To learn more about the Canvas panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+{{< video-embed src="/static/img/docs/canvas-panel/canvas-icon-value-mapping-support-9-3-0.mp4" max-width="750px" caption="Canvas panel icon value mapping support" >}}
+
+## Public dashboards improvements
+
```
```suggestion

We've made the following improvement to public dashboards.

```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Available in **experimental** in Grafana Open Source, Enterprise, and Cloud Advanced
+
+You can use Public Dashboards to make a given dashboard available to anyone on the internet with no need to sign in. In Grafana v9.3 we have introduced a new screen where you can manage all of your public dashboards. From here, you can view is a list of all of the public dashboards in your Grafana instance, navigate to the underlying dashboard, see if it is enabled, link out to the public version ...
```
```suggestion
You can use Public Dashboards to make a given dashboard available to anyone on the internet without needing to sign in. In Grafana v9.3, we have introduced a new screen where you can manage all of your public dashboards. From here, you can view a list of all of the public dashboards in your Grafana instance, navigate to the underlying dashboard, see if it is enabled, link out to the public version of the dashboard, or update the public dashboard's configuration. You can see a public dashboard's configuration if you have view access to the dashboard itself, and you can edit its configuration if you have the Admin or Server Admin role or the "Public Dashboard writer" role if you are using RBAC in Grafana Enterprise or Cloud Advanced.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+You can use Public Dashboards to make a given dashboard available to anyone on the internet with no need to sign in. In Grafana v9.3 we have introduced a new screen where you can manage all of your public dashboards. From here, you can view is a list of all of the public dashboards in your Grafana instance, navigate to the underlying dashboard, see if it is enabled, link out to the public version ...
+
+To check out this new screen and configure your public dashboards, navigate to Dashboards → Public Dashboards. 
```
```suggestion
To check out this new screen and configure your public dashboards, navigate to **Dashboards > Public Dashboards**. 
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+To check out this new screen and configure your public dashboards, navigate to Dashboards → Public Dashboards. 
+
+[image public-dashboard-audit-table.png]
```
Are we adding an image here?

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Annotations are now supported in public dashboards, with the exception of query annotations. They are turned off by default, but can be turned on in your public dashboard settings.
+
+Note that since Public Dashboards is an experimental feature, you need to enable it in Grafana using the `publicDashboards` [feature toggle]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}), or open a support ticket requesting public dashboards if you are a Cloud Advanced customer.
```
```suggestion
Note that because Public Dashboards is an experimental feature, you need to enable it in Grafana using the `publicDashboards` [feature toggle]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}), or open a support ticket requesting public dashboards if you are a Cloud Advanced customer.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Note that since Public Dashboards is an experimental feature, you need to enable it in Grafana using the `publicDashboards` [feature toggle]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}), or open a support ticket requesting public dashboards if you are a Cloud Advanced customer.
+
+Learn more about Public Dashboards in our [documentation]({{< relref "../dashboards/dashboard-public/" >}}).
```
```suggestion
To learn more about public dashboards, refer to [Public dashboards]({{< relref "../dashboards/dashboard-public/" >}}).
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+{{< figure src="/static/img/docs/enterprise/reports/report-zoom.png" max-width="750px" caption="Report zoom feature with PDF documents at three different zoom levels" >}}
+
+## Users and access
+
```
```suggestion

We've made the following improvements to users and access.

```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Generally available in all editions of Grafana
+
+As part of our efforts to improve the security of Grafana, we are introducing a long-awaited feature which enhances Grafana's OAuth 2.0 compatibility. When a user logs in using an OAuth provider, Grafana verifies on each request that the user's access token has not expired. When an access token expires, Grafana uses the refresh token provided (if any exists) to obtain a new access token.
```
```suggestion
As part of our efforts to improve the security of Grafana, we are introducing a long-awaited feature that enhances Grafana's OAuth 2.0 compatibility. When a user logs in using an OAuth provider, Grafana verifies on each request that the user's access token has not expired. Grafana uses the refresh token provided (if any exists) when an access token expires to obtain a new access token.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+### Resolve user conflicts in Grafana's CLI
+
+In older version of Grafana, usernames were case sensitive. This created conflicts, where a user might sign in using two differnt methods (like SAML and oauth) and have two accounts created, like `elastigirl@incredibles.com` and `ElastiGirl@incredibles.com`. Users in this situations might think they have lost their preferences and permissions. If this has occurred in your Grafana instance, you ca ...
```
```suggestion
In the older versions of Grafana, usernames were case-sensitive. This created conflicts, where a user might sign in using two different methods (like SAML and OAuth) and have two accounts created, like `elastigirl@incredibles.com` and `ElastiGirl@incredibles.com`. Users in this situation might think they have lost their preferences and permissions. If this has occurred in your Grafana instance, you can use a new Grafana CLI command to resolve user identity conflicts between users within Grafana.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+[This endpoint]({{< relref "../developers/http_api/access_control/#list-your-permissions" >}}) allows users to get the
+full list of RBAC permissions associated with their token.
+
+For more details, see the related service accounts [documentation]({{< relref
```
```suggestion
For more details, refer to [Debug the permissions of a service account token]({{< relref
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Generally available in Grafana Enterprise and Cloud Advanced
+
+All Grafana users can now use the latest release of [Terraform's Grafana provider](https://registry.terraform.io/providers/grafana/grafana/latest/docs) (version 1.31.1+) to provision [user and team access to service accounts](https://grafana.com/docs/grafana/latest/administration/service-accounts/#manage-users-and-teams-permissions-for-a-service-account-in-grafana). This allows full management of ...
```
```suggestion
All Grafana users can now use the latest release of [Terraform's Grafana provider](https://registry.terraform.io/providers/grafana/grafana/latest/docs) (version 1.31.1+) to provision [user and team access to service accounts]({{< relref "../administration/service-accounts/#manage-users-and-teams-permissions-for-a-service-account-in-grafana" >}}).

This allows full management of service accounts through Terraform - from creating a service account and allowing users to access it to assigning roles to the service account and generating service account tokens.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+All Grafana users can now use the latest release of [Terraform's Grafana provider](https://registry.terraform.io/providers/grafana/grafana/latest/docs) (version 1.31.1+) to provision [user and team access to service accounts](https://grafana.com/docs/grafana/latest/administration/service-accounts/#manage-users-and-teams-permissions-for-a-service-account-in-grafana). This allows full management of ...
+
+Grafana Enterprise and Cloud Pro and Advanced users can now provision [access to data sources](https://grafana.com/docs/grafana/latest/administration/data-source-management/#data-source-permissions) for Grafana's `Viewer`, `Editor` and `Admin` basic roles, as well as assign `Edit` permission.
```
```suggestion
Grafana Enterprise and Cloud Pro and Advanced users can now provision [access to data sources]({{< relref "../administration/data-source-management/#data-source-permissions" >}}) for Grafana's `Viewer`, `Editor`, and `Admin` basic roles, as well as assign `Edit` permission.
```

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,244 @@
+
+Grafana Enterprise and Cloud Pro and Advanced users can now provision [access to data sources](https://grafana.com/docs/grafana/latest/administration/data-source-management/#data-source-permissions) for Grafana's `Viewer`, `Editor` and `Admin` basic roles, as well as assign `Edit` permission.
+
+We have also added [documentation on provisioning RBAC roles and role assignments](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/rbac-terraform-provisioning/) to guide our Grafana Enterprise and Cloud Pro and Advanced users through this process.
```
```suggestion
We have also added [documentation on provisioning RBAC roles and role assignments]({{< relref "../administration/roles-and-permissions/access-control/rbac-terraform-provisioning/" >}}) to guide our Grafana Enterprise and Cloud Pro and Advanced users through this process.
```

nmarrs committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

nmarrs committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

nmarrs committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

nmarrs committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

nmarrs committed:
Update docs/sources/whatsnew/whats-new-in-v9-3.md

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

nmarrs committed:
Apply suggestions from code review

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

nmarrs committed:
Fix prettier

nmarrs committed:
Merge remote-tracking branch 'origin' into docs/whats-new-in-9.3

mjseaman committed:
Update whats-new-in-v9-3.md

mjseaman committed:
update nav wording

zsofiakm committed:
Screenshots for navigation and internazionalization

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,254 @@
+weight: -33
+---
+
+# What’s new in Grafana v9.3 (Beta)
```
```suggestion
# What’s new in Grafana v9.3
```

chri2547 committed:
adds alerting images

chri2547 committed:
fixes path to new nav and localization screenshots

mjseaman commented on docs/sources/whatsnew/whats-new-in-v9-3.md:
```diff
@@ -0,0 +1,258 @@
+
+We've improved the design and functionality of email templates to make template creation much easier and more customizable. The email template framework utilizes MJML to define and compile the final email HTML output. Sprig functions in the email templates provide more customizable template functions.
+
+{{< figure src="/static/img/docs/alerting/alert-templates-whats-new-v9.3.png" max-width="750px" caption="Email template redesign" >}}
```
```suggestion
{{< figure src="/static/img/docs/alerting/alert-templates-whats-new-v9.3.png" max-width="600px" caption="Email template redesign" >}}
```

mjseaman commented:
@chri2547 two comments: 

- The video for the canvas panel doesn’t render in my local version of the docs. Is that just a function of the locally-running version?
- The Email Templating screenshot is very long. I would suggest cropping or at least resizing it. See suggestion above.

GrafanaWriter reviewed (approved):
Thank you.
